walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -1,27 +1,88 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" dir="ltr">
   <head>
-    <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <meta charset="utf-8">
     <link rel="stylesheet" href="style.css">
+    <title>Raymond's Portfolio</title>
   </head>
   <body>
-    <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+    <div class="header-container">
+        <a class="header-item" href="#home">Raymond C. Alvarez Velez</a>
+        <a class="header-item" href="#about">About</a>
+        <a class="header-item" href="#work">My Work</a>
+        <a class="header-item" href="#contact">Contact</a>
+    </div>
+    <div class="content-container">
+        <div class="about-container">
+            <div class="image">
+              <img src="images/stanley-11.jpg" alt="">
```

Comment:
Branch RCAVelez-redesignIndex has addressed these issues, once that branch is approved I will rebase and merge. But this problem will propagate through each branch for now because of the timing of PRs